conftest.py

```
@@ -5,16 +5,29 @@
+import pytest
 from fake import FILE_REGISTRY
 from pytest_codeblock.constants import CODEBLOCK_MARK
 
 __author__ = "Artur Barseghyan <artur.barseghyan@gmail.com>"
 __copyright__ = "2023-2025 Artur Barseghyan"
 __license__ = "MIT"
-__all__ = ("pytest_runtest_teardown",)
+__all__ = (
+    "pytest_runtest_teardown",
+    "Optional",
+)
```

Ruff (0.15.6)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)